keras_cv/models/csp_darknet_test.py

```diff
@@ -29,6 +29,8 @@
 class CSPDarkNetTest(ModelsTest, tf.test.TestCase, parameterized.TestCase):
     @parameterized.parameters(*MODEL_LIST)
     def test_application_base(self, app, _, args):
+        if hasattr(tf.keras.__internal__, "enable_unsafe_deserialization"):
```

Comment:
This is a new internal API to allow the deserialization of lambdas (we are trying to actively discourage this, with exceptions such as where it's necessary in KerasCV). It is used in this test specifically because it is the only test in the suite that checks for serialization and deserialization.

The conditional is to ensure backwards-compatibility, since this new API only exists in Nightly now and will be released in the next stable release. We want to ensure it will work for users regardless of TF version.

Reply:
Got it -- thanks Neel!